homeassistant/components/octoprint/__init__.py

```diff
@@ -224,7 +224,7 @@ async def _async_update_data(self):
             printer = await self._octoprint.get_printer_info()
         except PrinterOffline:
             if not self._printer_offline:
-                _LOGGER.error("Unable to retrieve printer information: Printer offline")
+                _LOGGER.info("Unable to retrieve printer information: Printer offline")
```

Comment:
According the docs for integration, you are suppose to log warning messages on disconnect: 

> Handles device/service unavailable. Log a warning once when unavailable, log once when reconnected.

https://developers.home-assistant.io/docs/integration_quality_scale_index

Reply:
This is a strange one as this message says HA can connect to ocotprint, but the octoprint server isn't connected to the printer. Agreeed, warning does sound more appropriate than error though.

Reply:
So, the requirement from the QA scale index doesn't apply here, as we are connected to Octoprint.

We don't warn the logs either if a Philips Hue bulb goes offline on your printer.
IMHO, this should be logged as debug level. The fact the printer is offline should be reflected through the state of the entities, not the logs.

Reply:
```suggestion
                _LOGGER.debug("Unable to retrieve printer information: Printer offline")
```